pkg/component/monitoring/charts/bootstrap/templates/alertmanager/alertmanager.yaml

```diff
@@ -1,42 +1,4 @@
 {{ if .Values.alertmanager.enabled }}
-apiVersion: v1
-kind: Service
-metadata:
-  name: alertmanager-client
-  namespace: {{ .Release.Namespace }}
-  labels:
-    component: alertmanager
-    role: monitoring
-  annotations:
-    networking.resources.gardener.cloud/from-all-seed-scrape-targets-allowed-ports: '[{"port":9093,"protocol":"TCP"}]'
-    networking.resources.gardener.cloud/namespace-selectors: '[{"matchLabels":{"gardener.cloud/role":"shoot"}}]'
-spec:
-  ports:
-  - port: 9093
-    name: metrics
-  type: ClusterIP
-  selector:
-    component: alertmanager
-    role: monitoring
----
-apiVersion: v1
-kind: Service
-metadata:
-  name: alertmanager
-  namespace: {{ .Release.Namespace }}
-  labels:
-    component: alertmanager
-    role: monitoring
-spec:
-  ports:
-  - port: 9093
-    name: cluster
-  type: ClusterIP
-  # This is important! Without it the mech won't work.
-  clusterIP: None
-  selector:
-    component: alertmanager
-    role: monitoring
```

Comment:
Did I get your commit comment right that this will be created/managed by `prometheus-operator` now? I ask as only the `*-client` service is being managed by `gardenlet` now.

Reply:
Yes, the `prometheus-operator` creates a `Service` for the "mech"/peers (even though we don't need/use it since we only run one replica).